lib/core/batchingReceiver.ts

```diff
@@ -76,15 +76,8 @@ export class BatchingReceiver extends MessageReceiver {
 
     this.isReceivingMessages = true;
     return new Promise<ServiceBusMessage[]>((resolve, reject) => {
-      let onReceiveMessage: OnAmqpEventAsPromise;
-      let onSessionClose: OnAmqpEventAsPromise;
-      let onReceiveClose: OnAmqpEventAsPromise;
-      let onReceiveError: OnAmqpEvent;
-      let onSessionError: OnAmqpEvent;
-      let onSettled: OnAmqpEvent;
```

Comment:
Due to these tslint errors, declared them here.
```
ERROR: /home/travis/build/Azure/azure-service-bus-node/lib/core/batchingReceiver.ts[92, 71]: variable 'onReceiveError' used before declaration
ERROR: /home/travis/build/Azure/azure-service-bus-node/lib/core/batchingReceiver.ts[93, 65]: variable 'onReceiveMessage' used before declaration
ERROR: /home/travis/build/Azure/azure-service-bus-node/lib/core/batchingReceiver.ts[160, 69]: variable 'onSessionError' used before declaration
```

Reply:
Yup, just saw that. Fixed